src/Spice86.Core/Emulator/InterruptHandlers/Dos/Ems/EmmHandle.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace Spice86.Core.Emulator.InterruptHandlers.Dos.Ems;
+
+/// <summary>
+/// Represents a handle for allocated EMM memory to a DOS program.
+/// </summary>
+public class EmmHandle {
+    /// <summary>
+    /// The EMM handle number.
+    /// </summary>
+    public ushort HandleNumber { get; set; } = ExpandedMemoryManager.EmmNullHandle;
+    
+    private const string NullHandleName = "";
+    
+    /// <summary>
+    /// Gets or sets the handle name.
+    /// </summary>
+    public string Name { get; set; } = NullHandleName;
+
+    /// <summary>
+    /// The logical pages assigned to the handle.
+    /// </summary>
+    public IDictionary<ushort, EmmPage> PageMap { get; } = new Dictionary<ushort, EmmPage>();
```

Comment:
I don't see why this should be a Dictionary rather than a `List<EmmPage>`.

Reply:
The dictionary helped me catch when the code wasn't setting the PageNumber correctly.
The PageNumber is both the key of each item, and a property of the EmmPage class.
But yes, unless performance is a concern, I can replace this with an IList<EmmPage>

Reply:
Replaced with a list.